nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/distance_traveled_condition.hpp

```diff
@@ -79,4 +79,4 @@ class DistanceTraveledCondition : public BT::ConditionNode
 
 }  // namespace nav2_behavior_tree
 
-#endif  // NAV2_BEHAVIOR_TREE__PLUGINS__CONDITION__DISTANCE_TRAVELED_CONDITION_HPP_
+#endif  // NAV2_BEHAVIOR_TREE__PLUGINS__CONDITION__DISTANCE_TRAVELED_CONDITION_HPP_
```

Comment:
This EOL missing is going to cause a linting error, make sure to check the CI output or run `ament_cpplint` and `ament_uncrustify` locally to fix any such issues

Reply:
Looking over the code changes, there are definitely alot of linting errors, so you should definitelly run those locally to fix them all

Reply:
Fixed the current linting errors.